repoaudit/repoaudit/__init__.py

```diff
@@ -34,6 +34,38 @@
     )
 )
 
+apt_sources_option = click.option(
```

Comment:
If you're not planning on reusing this option, I would not bother to define a variable. I would just put it above the apt command.